mkdocs.yml

```diff
@@ -179,6 +179,7 @@ nav:
       - Developer Quickstart: Tutorials/Developer-Quickstart.md
     - Concepts:
         - Architecture: Concepts/ArchitectureOverview.md
+        - Bonsai Tries data storage format: Concepts/Bonsai.md
```

Comment:
I assume users can now select whether to use bonsai or forest as the data storage format (bonsai does not replace forest).

So should this concept topic rather be called "Data storage formats", and then we can describe the two formats here so that the user can make an informed decision about which storage format to use?

We should probably also have a HowTo topic e.g "Configure the data storage format" which will link to the concepts info.